src/src.pro

```diff
@@ -4,7 +4,7 @@
 
 TEMPLATE = app
 TARGET   = Mapper
-CONFIG  += c++11
+CONFIG  += c++14
```

Comment:
True, but we must fix src/src.pro.in first, src.pro is generated from it when CMake runs.